### PR TITLE
Add remote reuse pipeline & fix reuse mode bug

### PR DESCRIPTION
**Pull request by SparkSnail**
1. add remote reuse mode pipeline test files
2. when kill trial in reuse mode, the trial process might has finished, and kill process failed. Need to catch exception.

### CODE REVIEW

File: test/config/training_service.yml [hunk 1]
```diff
@@ -95,6 +95,8 @@ pai:
     containerNFSMountPath: 
     paiStorageConfigName: 
 remote:
+  remoteConfig:
+    reuse: false
```
Comment (QuanluZhang) [inline test/config/training_service.yml:99]: so there is no test for "reuse" is true?

Comment (SparkSnail) [reply test/config/training_service.yml:99]: no, the default value is false, and this value will set to true when pipeline set `$(remote_reuse)`.

**Comment by chicm-ms** [comment]
why do we add a pipeline for reuse?  can we just add a few `reuse` mode test cases instead?

**Comment by SparkSnail** [comment]
> why do we add a pipeline for reuse? can we just add a few `reuse` mode test cases instead?

1. To keep align with pai-reuse mode pipeline. 2. Add a pipeline can test all kinds of cases, only add a few reuse test cases might miss some scenario, for example this reuse pipeline tested that early stopped job might cause exception in reuse mode.

**Pull request closed without merging**